featurebyte/service/feature_readiness.py

```diff
@@ -206,7 +206,9 @@ async def _get_default_feature(self, feature_ids: Sequence[ObjectId]) -> Feature
         ):
             if default_feature is None:
                 default_feature = feature
-            elif feature.readiness > default_feature.readiness:
+            elif FeatureReadiness(feature.readiness) > FeatureReadiness(default_feature.readiness):
```

Comment:
Curious why isn't the type of `feature.readiness` `FeatureReadiness` if feature is a `FeatureModel`?

Reply:
This is due to this settings: https://github.com/featurebyte/featurebyte/blob/d99f4b8fc9731c5ed4fc7e1ed458604f5122d71e/featurebyte/models/base.py#L140

If we set this value sets to `False`, we will need to specify `feature.readiness.value` when dumping it into mongo. That's one of the reason why it is set to `True`. However, it causes this side effect & unfortunately it is not covered by existing tests...

Reply:
Example to check:
```
from enum import Enum

from pydantic import BaseModel


class SomeEnum(str, Enum):
    a = "a"
    b = "b"


class ModelA(BaseModel):
    val: SomeEnum


class ModelB(BaseModel):
    val: SomeEnum

    class Config:
        use_enum_values = True


model_a = ModelA(val="a")
model_b = ModelB(val="a")
assert isinstance(model_a.val, SomeEnum)
assert not isinstance(model_b.val, SomeEnum) and isinstance(model_b.val, str)
```

Output:
```
In [3]: model_a.val
Out[3]: <SomeEnum.a: 'a'>

In [4]: model_b.val
Out[4]: 'a'
```